docs/cromwell_features/WorkflowCallback.md

```diff
@@ -0,0 +1,58 @@
+The workflow callback is a simple way to integrate Cromwell with an external system. When each workflow reaches a terminal
+state, Cromwell will attempt to POST a message to a provided URL (see below for schema of this message). 
+Messages are sent for root workflows only, not subworkflows. Callback status information, including success or failure, 
+will be recorded in workflow metadata with keys containing `workflowCallback`.
+
+### Configuration
+
+This feature will only be used if enabled via config. All config items except `enabled` are optional.
+
+```
+workflow-state-callback {
```

Comment:
Can we include where in the config file should this go? Top level? In `engine` or `service`? Somewhere else? (meanwhile, I'm off to look through the code to try to find out 😄 )

Reply:
Seems like the answer is: yes, top level